beacon-chain/rpc/eventsv1/events.go

```diff
@@ -188,8 +195,7 @@ func (s *Server) handleStateEvents(
 func (s *Server) streamData(stream ethpb.Events_StreamEventsServer, name string, data proto.Message) error {
 	returnData, err := anypb.New(data)
 	if err != nil {
-		log.WithError(err).Error("Could not parse request from pb")
-		return err
+		return status.Errorf(codes.Internal, "could not prepare return data: %v", err)
```

Comment:
It only makes sense to return `status.Error` from top-level functions.
```suggestion
		return err
```